src/garage/tf/algos/npo.py

```diff
@@ -1,11 +1,16 @@
 """Natural Policy Gradient Optimization."""
 # pylint: disable=wrong-import-order
+import collections
+
 from dowel import logger, tabular
 import numpy as np
 import tensorflow as tf
 
+from garage import log_performance, TrajectoryBatch
 from garage.misc import tensor_utils as np_tensor_utils
-from garage.tf.algos.batch_polopt import BatchPolopt
+from garage.np.algos import RLAlgorithm
+from garage.sampler import OnPolicyVectorizedSampler
+from garage.tf import _functions
```

Comment:
you should use `from garage.tf. import paths_to_tensors` to serve as a good example